<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>8</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Segoe UI&quot;;
background-color: rgb(24, 24, 24);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_inscription">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>133</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 20px;
	background-color: rgb(33, 185, 255);
	font-weight:  bold;
	font-size: 15px;
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	
	background-color: rgb(16, 124, 255);
}</string>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>540</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 16, 20);
font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_prenom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37,39,48);	
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right:20px;
	background-color: rgb(34,36,44);
	font-size: 20px;
}

QLineEdit:hover{
	background-color: rgb(48,50,62);
}

QLineEdit:focus{
	border: 2px solid rgb(65,70,155);
}</string>
    </property>
    <property name="placeholderText">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_nom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37,39,48);	
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right:20px;
	background-color: rgb(34,36,44);
	font-size: 20px;
}

QLineEdit:hover{
	background-color: rgb(48,50,62);
}

QLineEdit:focus{
	border: 2px solid rgb(65,70,155);
}</string>
    </property>
    <property name="placeholderText">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_login">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37,39,48);	
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right:20px;
	background-color: rgb(34,36,44);
	font-size: 20px;
}

QLineEdit:hover{
	background-color: rgb(48,50,62);
}

QLineEdit:focus{
	border: 2px solid rgb(65,70,155);
}</string>
    </property>
    <property name="placeholderText">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_password">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37,39,48);	
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right:20px;
	background-color: rgb(34,36,44);
	font-size: 20px;
}

QLineEdit:hover{
	background-color: rgb(48,50,62);
}

QLineEdit:focus{
	border: 2px solid rgb(65,70,155);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_email">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37,39,48);	
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right:20px;
	background-color: rgb(34,36,44);
	font-size: 20px;
}

QLineEdit:hover{
	background-color: rgb(48,50,62);
}

QLineEdit:focus{
	border: 2px solid rgb(65,70,155);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 24px;
	font-weight: bold;
	
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_back">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>470</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 20px;
	background-color: rgb(255, 33, 33);
	font-weight:  bold;
	font-size: 15px;
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	
	background-color: rgb(16, 124, 255);
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
